store/storetest/status_store.go

```diff
@@ -82,11 +82,11 @@ func testStatusStore(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	if result := <-ss.Status().Get(status.UserId); result.Err != nil {
-		t.Fatal(result.Err)
+	if _, err := ss.Status().Get(status.UserId); err != nil {
```

Comment:
I would expect here something like `if status, err := ss.Status().Get(status.UserId); err != nil {` and use the returned value in the comparison later

Reply:
I miss here the `status` variable assignation.